Building multiple packages dependent on pods in the same workspace fails

```
CMake Error at pods/cmake/pods.cmake:335 (add_custom_target):
  add_custom_target cannot create target "tmp" because another target with
  the same name already exists.  The existing target is a custom target
  created in source directory
  "/home/wxm/tmp/oh-distro-private/catkin_ws/src/oh_translators".  See
  documentation for policy CMP0002 for more details.
```

this works just fine for the catkin-tools workflow but breaks the current workflow. reason is that it automatically tries to create the tmp directory

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 2.8.12)
 project(pods)
 
 find_package(catkin REQUIRED)

--- a/cmake/pods.cmake
+++ b/cmake/pods.cmake
@@ -332,7 +332,7 @@ macro(pods_config_search_paths)
         
         # hack to force cmake always create install and clean targets 
         install(FILES DESTINATION)
-        add_custom_target(tmp)
+        add_custom_target(tmp_${POD_NAME})
 
         set(__pods_setup true)
     endif(NOT DEFINED __pods_setup)

